testing/database.rst

```diff
@@ -228,6 +228,8 @@ so, get the entity manager via the service container as follows::
             $this->entityManager = $kernel->getContainer()
                 ->get('doctrine')
                 ->getManager();
+                
+            self::ensureKernelShutdown();
```

Comment:
I don't understand the need, it seems to me that this is already done in `KernelTestCase::tearDown()`.

Reply:
`KernelTestCase::tearDown()` is called after the test is run.
As stated in the PR description, the kernel must be shutdown before creating a client (which is typically done during the test).

Reply:
But then this would mean that `$this->entityManager` is not coming from the same kernel that is used in the actual test. This might not be an issue with this concrete example, but promoting it as a best practice may lead to wrong results with other services.

Reply:
Yes, this fix looks wrong. I think we should update the example to initialize `$this->client` in the set up instead.

Reply:
Good idea, and we could add a comment stating that the kernel must not be booted again inside the tests.

Reply:
Good idea, and we could add a comment stating that the kernel must not be booted again inside the tests.